packages/client/src/forms/configuration/default.ts

```diff
@@ -4332,6 +4337,1326 @@ export const registerForms: IDefaultRegisterForms = {
+                required: false,
+                initialValue: '',
+                validate: [],
+                placeholder: formMessageDescriptors.formSelectPlaceholder,
+                options: [
+                  {
+                    value: 'MONOGAMY',
+                    label: formMessageDescriptors.monogamy
+                  },
+                  {
+                    value: 'POLYGAMY',
+                    label: formMessageDescriptors.polygamy
+                  }
+                ],
+                mapping: {
+                  mutation: {
+                    operation: 'sectionFieldToBundleFieldTransformer',
+                    parameters: ['typeOfMarriage']
+                  },
+                  query: {
+                    operation: 'bundleFieldToSectionFieldTransformer',
+                    parameters: ['typeOfMarriage']
+                  },
+                  template: {
+                    fieldName: 'typeOfMarriage',
+                    operation: 'selectTransformer'
+                  }
+                }
+              }
+            ]
+          }
+        ]
+      },
+      {
+        id: MarriageSection.WitnessOne,
+        viewType: 'form',
+        name: formMessageDescriptors.witnessName,
+        title: formMessageDescriptors.witnessTitle,
+        groups: [
+          {
+            id: 'witness-view-group',
+            fields: [
+              {
+                name: 'seperator',
+                type: 'SUBSECTION',
+                label: {
+                  defaultMessage: 'Witness 1',
+                  description: 'Witness 1 desc',
+                  id: 'form.field.label.witness1'
+                },
+                initialValue: '',
+                ignoreBottomMargin: true,
+                validate: [],
+                conditionals: []
+              },
+              {
+                name: 'witnessOneFirstNamesEng',
+                previewGroup: 'witnessOneNameInEnglish',
+                type: 'TEXT',
+                label: formMessageDescriptors.firstName,
+                maxLength: 32,
+                required: true,
+                initialValue: '',
+                validate: [
+                  {
+                    operation: 'englishOnlyNameFormat'
+                  }
+                ],
+                mapping: {
+                  template: {
+                    fieldName: 'witnessOneFirstNamesEng',
+                    operation: 'nameToFieldTransformer',
+                    parameters: ['en', 'witnessOneFirstName']
+                  },
+                  mutation: {
+                    operation: 'fieldToNameTransformer',
+                    parameters: ['en', 'witnessOneFirstName']
+                  },
+                  query: {
+                    operation: 'nameToFieldTransformer',
+                    parameters: ['en', 'witnessOneFirstName']
+                  }
+                }
+              },
```

Comment:
I think we can directly use "firstNames" to be the field name. As otherwise it won't go to the proper fhir-builder. Same for the other name fields and for witnessTwo as well